extensions/hoarder/package.json

```diff
@@ -3,52 +3,212 @@
   "name": "hoarder",
   "title": "Hoarder",
   "description": "A self-hostable bookmark-everything app (links, notes and images) with AI-based automatic tagging and full text search",
-  "icon": "hoarder-icon.png",
-  "author": "kdurek",
+  "icon": "logo-icon.png",
+  "author": "luolei",
+  "contributors": [
+    "kdurek"
+  ],
+  "categories": [
+    "Web"
+  ],
   "license": "MIT",
   "commands": [
     {
-      "name": "search",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Reply:
If possible, if not it's also okay

Reply:
I'd suggest using the new command structure to replace the original one. Here's why:

- The original extension used the search API as the main entry point
- The new version uses the list API as the primary interface, with search integrated into all list views as a feature
- The entry points have evolved from a single search to multiple functional views (lists, tags, etc.)
- This better reflects the current Hoarder API capabilities and provides a more intuitive user experience

I believe this change won't significantly impact existing users as they'll get a more comprehensive interface in return.What do you think about this approach?